user_service_basic.go

```diff
@@ -0,0 +1,95 @@
+package gimlet
+
+import (
+	"context"
+	"crypto/md5"
+	"fmt"
+	"net/http"
+
+	"github.com/pkg/errors"
+)
+
+type NaiveUser struct {
```

Comment:
can you use basic user here?

Reply:
It doesn't have a password field. I could
1. Leave this as is.
2. Embed a basic user inside a naive user.
3. Add a password to the basic user.

Reply:
I vote for 3

Reply:
Done.